app/scripts/lib/message-manager.js

```diff
@@ -218,6 +219,13 @@ export default class MessageManager extends EventEmitter {
    *
    */
   rejectMsg(msgId) {
+    this.metricsEvent({
+      event: 'Reject Sig Request',
```

Comment:
This property corresponds to the "name" option passed into `metricsEvent` in the front end metric event this replaces.

The value for that event was this:

```suggestion
      event: 'Cancel Sig Request Via Notification Close',
```

We should try not to change the event details so that we don't break any reports built on the existing events.

Reply:
It looks like some of the other event names have changed as well. We'll want to ensure they all match what they were before.

Reply:
The name of the metric changed because the behaviour changed, it will be fired in more cases and doesn’t represent the same thing happening anymore

For example, it will be fired on pressing cancel button, not only on window close